config

```diff
@@ -306,6 +306,10 @@
 # option (string value)
 #version=V
 
+# After checking for content in the statedir, check the statedir-cache before
+# downloading it over the network
+#statedir-cache=[PATH]
```

Comment:
I know this is late but this should have been `stated_cache` (underscore), we are only using words with underscore in the config file. We have code to convert that _ to - before using it in swupd. Since this has already merged I will submit a PR to change this.